workflow_ephys/pipeline.py

```diff
@@ -17,11 +19,7 @@
 
 lab.activate(db_prefix + 'lab')
 
-subject.activate(db_prefix + 'subject',
-                 add_objects={'Source': lab.Source,
-                              'Lab': lab.Lab,
-                              'Protocol': lab.Protocol,
-                              'User': lab.User})
+subject.activate(db_prefix + 'subject', required_module=__name__)
 
 
 @lab.schema
```

Comment:
Should `SkullReference` be defined in `@schema` rather than `@lab.schema`. It may be cleaner that way.